MVVM Module/Source/Feed UI/Controllers/FeedRefreshViewController.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  FeedRefreshViewController.swift
+//  MVVM
+//
+//  Created by Alex Thurston on 6/16/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import UIKit
+import FeedFeature
+
+final class FeedRefreshViewController: NSObject {
+	@IBOutlet var view: UIRefreshControl?
+	@IBOutlet var errorView: ErrorView?
```

Comment:
The setter for this property can be private to prevent clients from changing the instance:

```suggestion
	@IBOutlet private(set) var errorView: ErrorView?
```